checker/src/net/jcip/annotations/GuardedBy.java

```diff
@@ -25,7 +25,7 @@
      * The Java expressions that need to be {@link LockHeld}.
      *
      * @see <a
-     *      href="http://types.cs.washington.edu/checker-framework/current/checkers-manual.html#java-expressions-as-arguments">Syntax
+     *      href="http://types.cs.washington.edu/checker-framework/current/checker-framework-manual.html#java-expressions-as-arguments">Syntax
```

Comment:
As a general rule, you can keep code reviews more manageable by making changes like these directly.